PSICompile/ILCodeGen.cs

```diff
@@ -131,7 +147,7 @@ public class ILCodeGen : Visitor {
          Out ("    call string [System.Runtime]System.String::Concat (string, string)");
       else {
          string op = b.Op.Kind.ToString ().ToLower ();
-         op = op switch { "mod" => "rem", "eq" => "ceq", "lt" => "clt", _ => op };
+         op = op switch { "mod" => "rem", "eq" => "ceq", "lt" => "clt", "geq" => "clt ldc.i4.0 ceq", "leq" => "cgt ldc.i4.0 ceq", "gt" => "cgt", "neq" => "ceq ldc.i4.0 ceq", _ => op };

```

Comment:
Don't output multiple IL codes on the same like (like `clt ldc.i40 ceq`). Typically, there should be only one OP code per line.